### PR TITLE
Create hi.json

### PR DESCRIPTION
**Pull request by skalavala**
Added Hindi language support :)

### CODE REVIEW

File: translations/hi.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "en": "Hindi"
```
Comment (balloob) [inline translations/hi.json]: Please change "en" to "hi". Language names should also be written in the Hindi language.

**Comment by skalavala** [comment]
Thank you... I thought I missed something there.

File: translations/hi.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "hi": "Hindi"
```
Comment (c727) [inline translations/hi.json]: use native language name https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes

**Comment by skalavala** [comment]
Thanks.. good learning. I was about to add another language, this definitely helps doing the right way next time.

File: translations/hi.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "hi": "हिंदी"
+  },
+  "panel": {
+    "config": "विन्यास",
+    "states": "अवलोकन",
+    "map": "नक्शा",
+    "logbook": "कार्यपंजी",
+    "history": "हिस्टरी",
```
Comment (arsaboo) [inline translations/hi.json]: How about `इतिहास` for `history`

File: translations/hi.json [hunk 4]
```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "hi": "हिंदी"
+  },
+  "panel": {
+    "config": "विन्यास",
+    "states": "अवलोकन",
+    "map": "नक्शा",
+    "logbook": "कार्यपंजी",
```
Comment (arsaboo) [inline translations/hi.json]: How about `डायरी` for `logbook`?

File: translations/hi.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "hi": "हिंदी"
+  },
+  "panel": {
+    "config": "विन्यास",
+    "states": "अवलोकन",
```
Comment (arsaboo) [inline translations/hi.json]: `स्थिति` would be more appropriate here.

File: translations/hi.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+  "language": {
+    "hi": "हिंदी"
+  },
+  "panel": {
+    "config": "विन्यास",
```
Comment (arsaboo) [inline translations/hi.json]: Let us remove this string. I am not sure most people will be able to understand what `विन्यास` means and many other translations don't have it.

Comment (skalavala) [reply translations/hi.json]: All coming from Google Translate... I am not a Hindi pudit you can tell :)

Comment (balloob) [reply translations/hi.json]: Let's not use Google Translate. We should only get translations from native speakers.

**Pull request closed without merging**